lib/smack/SmackOptions.cpp

```diff
@@ -55,6 +55,9 @@ const llvm::cl::opt<bool> SmackOptions::FloatEnabled(
 const llvm::cl::opt<bool> SmackOptions::MemorySafety(
   "memory-safety", llvm::cl::desc("Enable memory safety checks"));
 
+const llvm::cl::opt<bool> SmackOptions::IntegerOverflow(
+  "integer-overflow", llvm::cl::desc("Enable checking of integer arithmetic"));
```

Comment:
Description string should be `Enable integer overflow checks` to make it consistent with memory safety.

Reply:
Done.